app/src/sandbox/tablePaged/DynamicFilters/DynamicFilters.tsx

```diff
@@ -0,0 +1,124 @@
+import React, { useCallback, useMemo, useState } from "react";
+import css from "./DynamicFilters.scss";
+import sortBy from "lodash.sortby";
+import { Button, FlexRow, PickerInput } from "@epam/promo";
+import { DataRowOptions, FilterConfig, FiltersConfig, getOrderBetween, ITableState, useArrayDataSource } from "@epam/uui-core";
+import { PickerTogglerProps } from "@epam/uui-components";
+import { DynamicFilter } from "./DynamicFilter";
+
+interface DynamicFiltersProps<TFilter extends Record<string, any>> {
+    filters: FilterConfig<TFilter>[];
+    tableState: ITableState["tableState"];
+    setTableState: ITableState["setTableState"];
+}
+
+const DynamicFiltersImpl = <TFilter extends Record<string, any>>(props: DynamicFiltersProps<TFilter>) => {
+    const { filters, tableState, setTableState } = props;
+    
+    const mapFilterToPickerValue = useCallback((f: FilterConfig<TFilter>) => {
+        return {
+            id: f.columnKey,
+            columnKey: f.columnKey,
+            title: f.title,
+            isDisabled: tableState.filtersConfig[f.columnKey]?.isAlwaysVisible,
```

Comment:
if it isAlwaysVisible visible it not disabled, it means that you can't remove it. Also do we need to to remap filter config, why we can't just pass it as is to the filter?